codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -646,9 +647,10 @@ private LastHttpContent readTrailingHeaders(ByteBuf buffer) {
                         b.append(currentLastPos)
                          .append(lineTrimmed);
                         current.set(lastPos, b.toString());
-                    } else {
-                        // Content-Length, Transfer-Encoding, or Trailer
                     }
+                    //else {
+                    // Content-Length, Transfer-Encoding, or Trailer
```

Comment:
Thanks for listening @normanmaurer and @Scottmitch.

I understand where you're coming from @Scottmitch and I agree with your tacit point that this code should not have been changed in this PR in the first place as it has nothing to do with the stated purpose of the PR. My point is that for both contributors and maintainers a more optimal response would have been a request to revert and the creation of a new issue.